src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -403,6 +390,41 @@ public void afterNavigateTo(String url, WebDriver driver) {
     logJSError();
   }
 
+  /**
+   * All of tests should be executed as an user who opted in (agreed) on using ads tracking,
+   * unless you change it in @Execute annotation.
+   * Manually user would need to click 'agree' in the tracking opt in modal.
+   */
+  private void handleGdprModalWithCookie() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "handleGdprModalWithCookie" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)